python/fedml/computing/scheduler/model_scheduler/master_job_runner.py

```diff
@@ -460,9 +456,9 @@ def cleanup_runner_process(self, run_id):
     def start_device_inference_gateway(inference_port=ServerConstants.MODEL_INFERENCE_DEFAULT_PORT):
         # start unified inference server
         python_program = get_python_program()
-        master_port = os.getenv("FEDML_MASTER_PORT", None)
-        if master_port is not None:
-            inference_port = int(master_port)
+        master_port_frm_env = os.getenv(ServerConstants.ENV_MASTER_INFERENCE_PORT_KEY, None)
```

Comment:
Cool! Will add the necessary env variables and figure out sending from third party gpu resources.